### PR TITLE
bugfix async detection

### PR DESCRIPTION
**Pull request by takion**
The async method detection does'nt work in many case, I think this depends on js compilator and can't be trusted, by example, I use Babel and the stringification of method doesn't contains "async"

### CODE REVIEW

**Comment by f** [comment]
Then `super event, handler` needs to be removed.

**Comment by takion** [comment]
I just fixed it

File: README.md [hunk 1]
```diff
@@ -143,10 +143,10 @@ console.log(process.argv)
 
 ### Async API ⏩
 
-Omelette allows you to use `async` functions. You have to pass `Promise` object to the `reply` function.
+Omelette allows you to use `async` functions. You have to use onAsync and to pass `Promise` object to the `reply` function.
```
Comment (f) [inline README.md]: Please mark onAsync as code using backticks: `onAsync`

**Comment by takion** [comment]
done

**Comment by f** [comment]
Also there are some `async` examples on README, may you fix them too?

**Comment by takion** [comment]
Sorry, I missed one, fixed

**Pull request merged**